From 9e1421856e2a21134ba1beedc343a00017a8380e Mon Sep 17 00:00:00 2001
From: Noam Lessner <nlessner@habana.ai>
Date: Sat, 08 Feb 2020 20:43:31 +0200
Subject: [PATCH] [SW-8229] goya2: check fuse before recovery

We need to check efuse before recovery so we disable
debug I/F before we might enable PCIe link in recovery.

Change-Id: I22e06439943c84497dad37946abe43fdc99e1538
Signed-off-by: Noam Lessner <nlessner@habana.ai>
---

diff --git a/app/arcboot_rom.c b/app/arcboot_rom.c
index 77c850a..46b092b 100644
--- a/app/arcboot_rom.c
+++ b/app/arcboot_rom.c
@@ -245,8 +245,8 @@
 int arcboot_rom_main(void)
 {
 	arcboot_rom_init();
-	arcboot_rom_in_recovery();
 	arcboot_rom_fuse();
+	arcboot_rom_in_recovery();
 	arcboot_rom_bringup_pcie();
 	arcboot_rom_bringup_arcboot_ram();
 
